runtime/autoload/man.vim

```diff
@@ -402,6 +402,13 @@ function! s:format_candidate(path, psect) abort
 endfunction
 
 function! man#init_pager() abort
+  if exists('b:man_pager')
+    " Already inited, occurs as init_pager is called in syntax/man.vim
+    " and ftplugin/man.vim
+    return
+  endif
+  let b:man_pager = 1
```

Comment:
Since `b:man_pager` is local to buffer, I guess an error `E444: Cannot close last window` will occur when pressing `q` after jumping to another man page. Should it be changed to `w:man_pager` instead?